apps/challenges/views.py

```diff
@@ -446,6 +451,10 @@ def get_all_challenges(request, challenge_time):
         result_page, many=True, context={"request": request}
     )
     response_data = serializer.data
+    Metrics.total_http_requests_sample.labels(
+        "challenge", "get_all_challenges", request.method, "200"
+    ).inc()
```

Comment:
Also, we need to use 1 counter for each API and use tags in prometheus to distinguish them. This allows easy grouping and queries on a metric